### PR TITLE
Delete documents for Tizen Iot Preview 2

### PR DESCRIPTION
**Pull request by journey2w**
### Change Description ###

IoT team requested to delete Tizen IoT Preview 2.

### CODE REVIEW

**Comment by journey2w** [comment]
@jiehwan
Please see and check this updates. 
Thanks.

**Comment by journey2w** [comment]
Removing ARTIK-related pages will be decided next week.

**Comment by jiehwan** [comment]
in main page of https://docs.stg.tizen.org/staging/668/iot/
1. Things SDK API (https://docs.stg.tizen.org/staging/668/iot/guides/things-api) has broken link.. 
2. in left side navigator, sub menu of "Get Started > Setting up Board " has invalid link.

in page of https://docs.stg.tizen.org/staging/668/iot/get-started/overview
3. "Platform image / Headed" has no reference devices so it should be removed with ARTIK.

in page of "https://docs.stg.tizen.org/staging/668/iot/get-started/rpi3-5.0#flashing-tizen-images-with-iot-setup-manager"
4. in Overall, replace "IOT" to "IoT".
5. in "Installing Tizen Studio" section,
	a. in "3. Select the Extension SDK...", there is no need to install IOT-Headed-5.0 package.
6. in "Prerequisites" section,
	a. remove unnecessary contents of  "Platform image > Heades.... & Customized..."
7. Some figure displays the sample proile as artik_profile. it should be changed as rpi3.

**Comment by jiehwan** [comment]
@journey2w please check the review comments.

**Comment by journey2w** [comment]
> in main page of https://docs.stg.tizen.org/staging/668/iot/
> 
> 1. Things SDK API (https://docs.stg.tizen.org/staging/668/iot/guides/things-api) has broken link..
> 2. in left side navigator, sub menu of "Get Started > Setting up Board " has invalid link.
> 
> in page of https://docs.stg.tizen.org/staging/668/iot/get-started/overview
> 3. "Platform image / Headed" has no reference devices so it should be removed with ARTIK.
> 
> in page of "https://docs.stg.tizen.org/staging/668/iot/get-started/rpi3-5.0#flashing-tizen-images-with-iot-setup-manager"
> 4. in Overall, replace "IOT" to "IoT".
> 5. in "Installing Tizen Studio" section,
> a. in "3. Select the Extension SDK...", there is no need to install IOT-Headed-5.0 package.
> 6. in "Prerequisites" section,
> a. remove unnecessary contents of "Platform image > Heades.... & Customized..."
> 7. Some figure displays the sample proile as artik_profile. it should be changed as rpi3.

Fixed except figures. Developers have to provide new figures.

File: docs/iot/get-started/overview.md [hunk 1]
```diff
@@ -4,25 +4,23 @@ Tizen is commercialized for smart TV, smart phone, smart watch, and smart home a
...
 Tizen IoT Extension SDK version 1.0 is released from Tizen Studio 3.0 for Tizen 5.0. The following features are supported:
 
--   Hardware targets: ARTIK 530 and ARTIK 530s are supported as the official reference hardwares.Raspberry Pi 3 is also supported
+-   Hardware targets: Raspberry Pi 3 is supported as the official reference hardwares.
 
 -   Platform image
-    -   Headless: Tizen 5.0 platform image for headless-type IoT devices without a display. Raspberry Pi 3, ARTIK 530, and ARTIK 530s can be used with the headless platform image.
-
-    -   Headed: Tizen 5.0 platform image for headed-type IoT devices with a display. ARTIK 530 and ARTIK 530s can be used with the headed platform image.
+    -   Headless: Tizen 5.0 platform image for headless-type IoT devices without a display. Raspberry Pi 3 can be used with the headless platform image.
```
Comment (jhchoilee) [inline docs/iot/get-started/overview.md]: "Headed: Tizen 5.0 platform image for headed-type IoT devices with a display. Raspberry Pi 3 can be used with the headed platform image." --> Headed에서 Rpi3 지원합니다. 해당 문구가 포함될 수 있도록 해주세요

Comment (journey2w) [reply docs/iot/get-started/overview.md]: Done

**Pull request merged**